validation_server/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "propagation-tests",
+  "version": "1.0.0",
+  "description": "server for w3c tests",
+  "main": "validation_server.js",
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@opentelemetry/api": "^0.10.1",
+    "@opentelemetry/node": "^0.10.1",
+    "@opentelemetry/plugin-express": "^0.8.0",
```

Comment:
You are depending on a lot of autoinstrumentation modules for what should be a very basic test. I would skip everything except for `@opentelemetry/{api,core,tracing}`, and call the propagation and tracing apis manually.